New Copy & Run button styles implementation

cherifGsoul opened the pull request:
Fixes #561 
Depends on https://github.com/bit-docs/bit-docs-html-codepen-link/pull/14

<img width="881" alt="Screen Shot 2019-08-13 at 5 22 05 PM" src="https://user-images.githubusercontent.com/109013/62960440-8f92ab00-bdf2-11e9-8f18-9a1620b7f7c8.png">
<img width="895" alt="Screen Shot 2019-08-13 at 5 22 12 PM" src="https://user-images.githubusercontent.com/109013/62960442-902b4180-bdf2-11e9-8a00-9df52f05fd31.png">
<img width="908" alt="Screen Shot 2019-08-13 at 5 22 16 PM" src="https://user-images.githubusercontent.com/109013/62960443-902b4180-bdf2-11e9-8e5c-cc8b4ccc0808.png">
<img width="888" alt="Screen Shot 2019-08-13 at 5 22 25 PM" src="https://user-images.githubusercontent.com/109013/62960445-902b4180-bdf2-11e9-93b9-df49af163bf3.png">
<img width="884" alt="Screen Shot 2019-08-13 at 5 22 38 PM" src="https://user-images.githubusercontent.com/109013/62960446-90c3d800-bdf2-11e9-8d3c-fa1695687cd3.png">
<img width="883" alt="Screen Shot 2019-08-13 at 5 22 52 PM" src="https://user-images.githubusercontent.com/109013/62960449-90c3d800-bdf2-11e9-9c68-59bbfda7462e.png">

phillipskevin commented on static/prism.less:
```diff
@@ -0,0 +1,41 @@
+		top: 0 !important;
+		right: 0 !important;
+		border-left: 1px #dfdfdf solid;
+    	border-bottom: 1px #dfdfdf solid;
```
Can you fix this indenting?

cherifGsoul replied:
Fixed, thanks!

chasenlehara commented on static/canjs.less:
```diff
@@ -22,3 +22,4 @@
 @import "screenshots.less";
 @import "loading-bar.less";
 @import "search.less";
+@import "prism.less";
```
How was this being loaded before?

chasenlehara commented on static/prism.less:
```diff
@@ -0,0 +1,41 @@
+	> .toolbar {
+		line-height: 0;
+		top: 0 !important;
+		right: 0 !important;
```
Are the `!important` necessary? If yes right now, can we use a more specific selector to override the other styles, or are there other styles that should be removed?

cherifGsoul replied:
Fixed, thanks

chasenlehara commented on static/prism.less:
```diff
@@ -0,0 +1,41 @@
+	color: #858585 !important;
+	padding: .35em 2.25em !important;
+	box-shadow: none !important;
+	border-radius: 0 !important;
```
Same thing for the `!important` in the rest of this file. We should avoid them if at all possible.

cherifGsoul commented:
> This doesn’t look quite right (buttons over the expand bar):

What is not right, the style or Run button is missed?

> I don’t see the Run button integrated into the toolbar. Should that be in this PR?

@chasenlehara did you update `bit-docs-html-codepen-link` to major? https://github.com/canjs/bit-docs-html-canjs/blob/5543af358184a36f2257c77f7ced9d1d9d6990b2/make-example.js#L31

cherifGsoul commented:
@chasenlehara I updated the styles, for notes, buttons toolbar is a sibling to the collapsible bar parent.
I updated the `make-example.js` with the last major of `bit-docs-html-codepen-link`
